templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+                    </div>
+                    <div class="col-5 u-align--center u-hide--small">
+                        {{ image (
+                            url="https://assets.ubuntu.com/v1/ac9a0e00-lxd_primary.svg",
+                            alt="",
+                            width="229",
+                            height="75",
+                            hi_def=True,
+                            loading="auto"
+                            ) | safe
+                        }}
+                    </div>
+                </div>
+                <div class="u-fixed-width">
+                    <hr class="p-separator">
+                </div>
+                <div class="row u-vertically-center">
+                    <div class="col-7">
+                        <h3>Software orchestration</h3>
+                        <p>Streaming Android apps to thousands of mobile and desktop clients simultaneously requires orchestration and monitoring of several software components. This involves streaming servers, image repositories, applications, a dashboard application for management and more. Thanks to Juju and MAAS, you can deploy your stack automatically anywhere. </p>
+                    </div>
+                    <div class="col-5 u-align--center u-hide--small">
+                        {{ image (
+                            url="https://assets.ubuntu.com/v1/48a8eddf-juju_black-orange_hex.svg",
+                            alt="",
+                            width="208",
+                            height="75",
+                            hi_def=True,
+                            loading="auto"
+                            ) | safe
+                          }}
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="p-strip">
+            <div class="row u-vertically-center">
+                <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+                        alt="",
+                        width="289",
+                        height="205",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+                </div>
+                <div class="col-5">
+                    <h3>Invent future mobile services</h3>
+                    <p>Get in touch with Canonical to explore avenues to deliver <br>new value-added services to millions via cloud streaming. </p>
+                    <button class="p-button--positive">Get in touch</button>
```

Comment:
this will need to be a link to "/contact-us". If you wait until this [PR](https://github.com/canonical-web-and-design/anbox-cloud.io/pull/166) is merged then the contact us page will be live

Reply:
The PR is merged now so you can link this to `/contact-us`